### ISSUE TITLE
[Accessibility]: 

### ISSUE THREAD

**Issue by jasongornall** [sourcegraph/sourcegraph#36251]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34431

### Problem description

![image](https://user-images.githubusercontent.com/6912393/171050382-1788354f-ef10-44e9-a1ab-092f269d0cc5.png)

Need to improve contract for red / green view and the gray text

### Expected behavior

A better contract score. Will need to work with design to identify a more compliant contrast.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
The work to fix this has been merged in https://github.com/sourcegraph/sourcegraph/pull/37385. It is currently feature flagged so we can test internally before enabling it site-wide.

We have an issue (https://github.com/sourcegraph/sourcegraph/issues/37535) to enable it fully over the coming weeks, closing this in favour of that!
